src/implementations/minecraft_bedrock/server.rs

```diff
@@ -0,0 +1,515 @@
+                                        instance_event_inner: InstanceEventInner::SystemMessage {
+                                            message: line,
+                                        },
+                                        instance_name: name.clone(),
+                                    }),
+                                    details: "".to_string(),
+                                    snowflake: Snowflake::default(),
+                                    caused_by: CausedBy::System,
+                                });
+                                if let Some((player_name, xuid)) = parse_player_joined(&system_msg) {
+                                    players_manager.lock().await.add_player(
+                                        MinecraftBedrockPlayer {
+                                            name: player_name.clone(),
+                                            uuid: Some(xuid.clone()),
+                                        },
+                                        self.name().await,
+                                    );
+                                } else if let Some(player_name) = parse_player_left(&system_msg) {
+                                    players_manager
+                                        .lock()
+                                        .await
+                                        .remove_by_name(&player_name, self.name().await);
+                                }
+                            }
+                        }
+                        info!("Instance {} process shutdown", name);
+                        self.state
+                            .lock()
+                            .await
+                            .try_transition(
+                                StateAction::InstanceStop,
+                                Some(&|state| {
+                                    self.event_broadcaster.send(Event {
+                                        event_inner: EventInner::InstanceEvent(InstanceEvent {
+                                            instance_name: config.name.clone(),
+                                            instance_uuid: self.uuid.clone(),
+                                            instance_event_inner:
+                                                InstanceEventInner::StateTransition { to: state },
+                                        }),
+                                        snowflake: Snowflake::default(),
+                                        details: "Instance stopping as server process exited"
+                                            .to_string(),
+                                        caused_by: cause_by.clone(),
+                                    });
+                                }),
+                            )
+                            .unwrap();
+                        self.players_manager.lock().await.clear(name);
+                    }
+                });
```

Comment:
remove this code since bedrock does not emit console messages

Reply:
You need another way to do tell if an instance has fully started

Reply:
bedrock does emit console messages though. screenshot from just running the exe:
![image](https://github.com/Lodestone-Team/lodestone_core/assets/29615067/b8a489f9-2cbb-4996-b2aa-294743df7a1c)

Reply:
In this case you can simply match for the "Server started" message and emit an instance state transition on that

Reply:
No need to parse for any other messages

Reply:
that's what it should be doing already unless i made a mistake in the implementations somewhere. i modified the console parsing regex to work for bedrock